media-sound/pulseaudio/pulseaudio-14.99.2_pre1.ebuild

```diff
@@ -213,9 +213,9 @@ multilib_src_configure() {
 		#$(meson_use cpu_flags_arm_neon neon-opt)
 		$(meson_native_use_feature tcpd tcpwrap)
 		$(meson_feature dbus)
-		$(meson_feature elogind)
+		$(meson_native_use_feature elogind)
 		$(meson_feature X x11)
-		$(meson_feature systemd)
+		$(meson_native_use_feature systemd)
```

Comment:
You should also change `systemd? ( sys-apps/systemd:=[${MULTILIB_USEDEP}] )` to `systemd? ( sys-apps/systemd:= )`.

Reply:
thanks, fixed this and dropped multilib requirement from `sys-apps/tcp-wrappers` as well